lib/auth/apiserver_test.go

```diff
@@ -129,7 +131,7 @@ func TestUpsertServer(t *testing.T) {
 			addServers(s.GetAuthServers())
 			addServers(s.GetNodes(defaults.Namespace))
 			addServers(s.GetProxies())
-			require.Empty(t, cmp.Diff(allServers, []services.Server{tt.wantServer}))
+			require.Empty(t, cmp.Diff(allServers, []services.Server{tt.wantServer}, cmpopts.IgnoreFields(types.Metadata{}, "ID")))
```

Comment:
Due to changes to memory backend, now it behaves similar to etcd and generates new IDs in `PutXXX` (see below).